src/D2L.Bmx/Okta/Models/AuthenticateResponse.cs

```diff
@@ -21,17 +21,68 @@ internal record AuthenticateResponseEmbedded(
 	OktaMfaFactor[]? Factors
 );
 
+[JsonPolymorphic(
+	TypeDiscriminatorPropertyName = "factorType",
```

Comment:
Unfortunately, STJ doesn't offer a way to include the property in JSON used for type discrimination in the de-serialized object.

I did find a issue asking for this capability (can't find it now). It makes sense since ideally if you are using type discrimination, then the rest of your code should use the C# type system to handle differences in the various types.

In this case, where we only want to treat one of several types differently (but include different names for each type), this is incredibly cumbersome.